enrichment-definition/src/main/java/eu/europeana/enrichment/model/impl/Keyword.java

```diff
@@ -0,0 +1,128 @@
+package eu.europeana.enrichment.model.impl;
+
+import java.util.List;
+
+import org.bson.types.ObjectId;
+
+import dev.morphia.annotations.Entity;
+import dev.morphia.annotations.Id;
+
+@Entity(value="Keyword")
```

Comment:
should use small cap for first letter, i.e. keyword, as this seems to be a requirement for spring data